planning/development-strategy.md

```diff
@@ -25,86 +25,161 @@ A simple little website about trees.
+  links(issue)
 
 ### Introduction: CSS
 
-- styling to make the text look fancy
+- styling to make the text look fancy.(issue)
 
 ---
 
-## Main Info
+## Content (milestone)
 
 > "As a user I can learn about trees"
 
-There should be enough information that the user can learn the basics about
-trees.
+Since it is a home page we don't place much content but there should be links to
+the content with clear description.
+
+- [ ] _A section with some general information about the uses of trees,tree
+      health and social impact_(issue)
+- [ ] _there are links to different sources providing content about trees_
+
+### Content: HTML
+
+- a div with title and paragraph explaining why Batman is the best
+  superhero(issue)
+
+### Content: CSS
+
+- [ ] hover effect on links (issue)
+- [ ] styling to make the content look nice(issue)
+
+## Extra Info (milestone)
+
+> "As a user I can learn even more about trees" Some more links for students who
+> want to go beyond the basics.
+
+- [ ] _there is an aside with links for further reading_
+
+### Extra Info: HTML
+
+- An aside with links for digging deeper, some helpful text and a reference link
+  (issue)
+
+### Extra Info: CSS
+
+- styling to position the aside (issue)
+- styling to style the text (issue)
+
+## Main Info (milestone)
+
+> "As a user I can learn about trees" There should be enough information that
+> the user can learn the basics about trees.
 
 - [ ] _there is a main text in the page with helpful info and links_
+- [ ] _there are two images in the left side of the main page_
 
 ### Main Info: HTML
 
-- Main text: a section with some general information about the project
-- Extra Info Links: a list of links
+- Main text: a section with some general information about the project.(issue)
+
+- Extra Info Links: a list of links (issue)
 
 ### Main Info: CSS
 
-- style the text
-- styling to format the list
+- style the text (issue)
+
+## Images (milestone)
+
+> "As a user I want to see the images of Tree's Health/social impact"
+
+There should be an image of Tree.
+
+- [ ] _there is an image explains about Tree health/ slcial impact_
+
+### Images: HTML
+
+- A div with an img of Tree (issue)
+
+### Images: CSS
+
+- styling the div to have a good width(issue)
+- style the image to be centered(issue)
 
 ---
 
-## Extra Info
+## Footer (milestone)
 
-> "As a user I can learn even more about trees"
+> "As a user I can learn if the page is reserved for a specific user and when it
+> has been created"
 
-Some more links for students who want to go beyond the basics.
+Some info about the creation of the website date and reserver right.
 
-- [ ] _there is an aside with links for further reading_
+- [ ] _there is info about the date which the website is created and reserved
+      right_
 
-### Extra Info: HTML
+### Footer: HTML
 
-- An aside with links for digging deeper, some helpful text and a reference link
+- A footer with a paragraph with the date of the page creation and the name of
```

Comment:
So you suggest to delete all contact links from footer?

Reply:
Yeah as of now...then later we have time we can add contact links

Reply:
But it's much easier just copy paste links as they are in the code than first delete them from code, then add "paragraph with the date of the page creation"...May be we should rebuild existing code first and then if we have time change it. Moreover it works with our User stoties

Reply:
ok then let me add contact links.